src/MongoDB.Driver.GridFS/GridFSBucketOptions.cs

```diff
@@ -99,6 +102,18 @@ public int ChunkSizeBytes
                 _chunkSizeBytes = value;
             }
         }
+        
+        /// <summary>
+        /// Gets or sets whether to disable MD5 checksum computation when uploading a GridFS file.
+        /// </summary>
+        /// <value>
+        /// Whether or not MD5 checksum computation is disabled when uploading a GridFS file.
```

Comment:
This text is out of sync with the summary (but only *very* slightly).

Doesn't need to be updated unless you want to.